internal/utils/bucket.go

```diff
@@ -12,3 +12,18 @@ func Bucketize(value uint, buckets []uint) string {
 	}
 	return ">" + strconv.Itoa(int(buckets[len(buckets)-1]))
 }
+
+// LinearBuckets returns an evenly distributed range of buckets in the closed interval
+// [min...max]. The min and max count toward the bucket count since they are included
+// in the range.
+func LinearBuckets(min, max float64, count int) []float64 {
```

Comment:
I looked into using https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#LinearBuckets but it doesn't allow me to define the closed interval I wish to build the bucket range around. It only allows me to specify a start and width and count of buckets. I wanted to define a start and end and count of buckets and distribute the buckets evenly around that close-interval range.

This is why I added this function, and the function name is inspired by the prometheus one.